apps/frontend/core-dapp/src/features/trade/TradeStore.ts

```diff
@@ -901,6 +903,19 @@ export class TradeStore {
     return this.root.depositStore.calculateDepositRewardValue(ppoRewardForOpenTrade)
   }
 
+  get depositLimit(): DepositLimit {
+    if (this.action !== 'open' || this.paymentToken.type !== 'native') {
```

Comment:
do we need this check? Don't think it's relevant here because withdraw flow doesnt use deposit limit
```suggestion
    if (this.paymentToken.type !== 'native') {
```

Reply:
Same logic as with [this](https://github.com/prepo-io/prepo-monorepo/pull/1115#discussion_r1147258469). I'm happy to remove it, but personally it would be helpful for me.